tabpfn_client/prompt_agent.py

```diff
@@ -126,7 +126,12 @@ def prompt_add_user_information(cls, user_auth_handler: "UserAuthenticationClien
         )
         contact_via_email = True if choice_contact == "y" else False
 
-        user_auth_handler.add_user_information(company, role, use_case, contact_via_email)
```

Comment:
you need to delete UserAuthenticationClient.add_user_information function still, right?